detox/src/devices/allocation/drivers/android/genycloud/GenyRecipeQuerying.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * @param {import('./services/GenyRecipesService')} recipesService
+ */
 class GenyRecipeQuerying {
```

Comment:
wrong jsdoc location - move it one line down

Reply:
LOL